CHANGELOG.md

```diff
@@ -188,6 +189,9 @@ binary | sha256 hash
 
 **No notable changes for this release**
 
+## Known issues
+
+* master newer than v1.4.0 has backward compatible issues with older kubectl. Specifically, if "kubectl rolling-update" does not change the name of the replication controller (RC), the command will fail to create the new RC and report the RC already exists. When this happens, the pods are successfully updated, but the name of the RC is its original name plus a hash value. If you want to rename the RC, please download a kubectl newer than v1.4.0, then run rolling-update to change the name.
```

Comment:
Suggest make a section or bullet point here to let people know it's kubectl rolling-update related and they can skip if they're not using it. Something like:

### kubectl rolling-update
blah blah 

or 

* **`kubectl rolling-update`**: blah blah